src/components/implementation/no_interface/cFE_booter/osfiles.c

```diff
@@ -26,70 +33,71 @@ int32 OS_creat(const char *path, int32  access)
 */
 int32 OS_open(const char *path,  int32 access,  uint32 mode)
 {
-    if (access != OS_READ_ONLY) PANIC("Read Only Filesystem!");
-    if (!path) return OS_FS_ERR_INVALID_POINTER;
-    if (strlen(path) > OS_MAX_PATH_LEN) return OS_FS_ERR_PATH_TOO_LONG;
-    return open(path);
+    int32 ret = chk_path(path);
```

Comment:
I'm pretty sure that even cFE doesn't want `verb_noun` variable/function names.  Please read the CSG.  This function should at least be `path_chk`, but more likely `fs_path_chk`.